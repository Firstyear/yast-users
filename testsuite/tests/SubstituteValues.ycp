/**
 * Test for UsersLDAP::SubstituteValues
 * Author	: Jiri Suchomel <jsuchome@suse.cz>
 * $Id$
 */
{
    // testedfiles: UsersLDAP.pm

    import "Testsuite";
    import "Mode";
    import "UsersLDAP";

    map<string,any> user = $[
	"homedirectory"	: "/home/%uid",
	"cn"		: "%uid",
	"uid"		: "hh",
    ];

    Mode::SetTest ("test");

    Testsuite::Dump (user);

    Testsuite::Test (``(UsersLDAP::SubstituteValues ("user", user)), [], 0);
}
