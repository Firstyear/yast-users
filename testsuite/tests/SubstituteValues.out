Dump	$["cn":"%uid", "homedirectory":"/home/%uid", "uid":"hh"]
Return	$["cn":"hh", "homedirectory":"/home/hh", "uid":"hh"]
